library/src/cache/updater_state.rs

```diff
@@ -73,6 +82,8 @@ impl UpdaterState {
             serialized_state: SerializedState {
                 release_version,
                 queued_events: Vec::new(),
+                // Generate random number in the range [1, 100].
+                rollout_group: rand::thread_rng().gen_range(1..101),
```

Comment:
Is 1...100 the same?

Reply:
1..=100  https://doc.rust-lang.org/std/ops/struct.RangeInclusive.html

Reply:
Done